keras_nlp/models/bert/bert_presets.py

```diff
@@ -12,13 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from collections import defaultdict
-
-# TODO(jbischof): document checkpoints in keras.io and use URL in docstrings
+# TODO(jbischof): document presets in keras.io and use URL in docstrings
 # Metadata for loading pretrained model weights.
-checkpoints = {
+presets = {
```

Comment:
Let's maybe call this `backbone_presets` for now? So we leave room for `classifier_presets` somewhere down the line. (Or we could keep a flat list and track some metadata)

For now this list is specifically a list of presets for the bert backbone, so let's name as such.

Reply:
We probably should have a separate dict for each task.